test/WebJobs.Script.Tests/Configuration/ExtensionBundleOptionsSetupTest.cs

```diff
@@ -0,0 +1,184 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using Microsoft.Azure.WebJobs.Script.BindingExtensionBundle;
+using Microsoft.Azure.WebJobs.Script.Configuration;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging;
+using Microsoft.WebJobs.Script.Tests;
+using Xunit;
+
+namespace Microsoft.Azure.WebJobs.Script.Tests.Configuration
+{
+    public class ExtensionBundleOptionsSetupTest
+    {
+        private readonly TestEnvironment _environment = new TestEnvironment();
+        private readonly TestLoggerProvider _loggerProvider = new TestLoggerProvider();
+        private readonly string _hostJsonFile;
+        private readonly ScriptApplicationHostOptions _options;
+
+        public ExtensionBundleOptionsSetupTest()
+        {
+            string rootPath = Path.Combine(Environment.CurrentDirectory, "ScriptHostTests");
+
+            if (!Directory.Exists(rootPath))
+            {
+                Directory.CreateDirectory(rootPath);
+            }
+
+            _options = new ScriptApplicationHostOptions
+            {
+                ScriptPath = rootPath
+            };
+
+            _hostJsonFile = Path.Combine(rootPath, "host.json");
+            if (File.Exists(_hostJsonFile))
+            {
+                File.Delete(_hostJsonFile);
+            }
+        }
+
+        [Theory]
+        [InlineData(@"{
+                    'version': '2.0',
+                    }")]
+        public void MissingExtenstionConfig_DoesNotThrowException(string hostJsonContent)
```

Comment:
Combine tests using memberdata similar to https://github.com/Azure/azure-functions-host/blob/dev/test/WebJobs.Script.Tests/Description/DotNet/DotNetFunctionInvokerTests.cs#L140

Reply:
Keeping this in place since the data with the tests would make it easier to debug the tests. And the test data is not re-used anywhere.